catalog/registry/catalog-registry-api-impl/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version. 
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
+
+    <OCD name="Registry Catalog Store" id="Csw_Registry_Store" description="Registry Catalog CSW Store">
+
+        <AD description="The unique name of the store" name="Registry ID" id="id" required="true"
+            type="String"/>
+
+        <AD description="URL to the endpoint implementing the CSW EBRIM registry spec"
```

Comment:
Based on our conversations, I don't think this is a true statement. We are utilizing the schemas from EBRIM but we are not implementing the EBRIM profile. We need to be clear that this is a custom implementation that is not meant to be compatible with other EBRIM CSW ISO Services.